core/src/toga/app.py

```diff
@@ -349,6 +348,29 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def paths(self):
+        """
+        Paths for known safe system locations. Using paths makes sure you have a consistent and known location to store
+        files for your app. Also, some platforms do not allow arbitrary file access to any location on disk.
+        On Android, there are many limitations when it comes to accessing file locations and the apis
```

Comment:
This isn't an Android-specific thing. iOS has the same set of constraints - and technically, so do macOS and Windows when you use packaged/sandboxed apps.

Also - it's not just that they're *known* locations - it's that they're *platform appropriate* locations.